pkg/option/config.go

```diff
@@ -1069,6 +1069,9 @@ const (
 
 	// ConnectivityProbeFrequencyRatio is the name of the option to specify the connectivity probe frequency
 	ConnectivityProbeFrequencyRatio = "connectivity-probe-frequency-ratio"
+
+	// DevicePrefixesToExclude excludes devices that have the prefixes.
+	DevicePrefixesToExclude = "device-prefixes-to-exclude"
```

Comment:
nit: With the new hive config struct it's no longer recommended to add the name of the flag in `pkg/option/config.go` (as it was preferred when using the global daemon config struct). These changes just trigger "unnecessary" code reviews (agent & cli).

I'm aware that the other flags of `DevicesConfig` are defined the same way, but i'd still recommend to inline this one.